internal/cloud/realm/secrets_ext_test.go

```diff
@@ -0,0 +1,60 @@
+package realm_test
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	u "github.com/10gen/realm-cli/internal/utils/test"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func TestSecrets(t *testing.T) {
+	u.SkipUnlessRealmServerRunning(t)
+
+	t.Run("should fail without an auth client", func(t *testing.T) {
+		client := realm.NewClient(u.RealmServerURL() + "")
+
+		_, err := client.Secrets(primitive.NewObjectID().Hex(), primitive.NewObjectID().Hex())
+		assert.Equal(t, realm.ErrInvalidSession{}, err)
+	})
+
+	t.Run("with an active session ", func(t *testing.T) {
+		client := newAuthClient(t)
+		groupID := u.CloudGroupID()
+
+		secretName := "secretName"
+		secretValue := "secretValue"
+
+		testApp, appErr := client.CreateApp(groupID, "secrets-test", realm.AppMeta{})
+		assert.Nil(t, appErr)
+		defer client.DeleteApp(groupID, testApp.ID)
+
+		assert.Nil(t, client.Import(groupID, testApp.ID, map[string]interface{}{
+			app.NameAuthProviders: []map[string]interface{}{
```

Comment:
[question] do you need this import call?  why are auth providers necessary to be present when working with app secrets?

Reply:
This was my bad from skimming over it while copy/paste lol. Removing